.build/parent-pom-template.xml

```diff
@@ -233,12 +233,52 @@
       <id>zznate</id>
       <name>Nate McCall</name>
     </developer>
+    <developer>
+      <id>smiklosovic</id>
+      <name>Stefan Miklosovic</name>
+    </developer>
   </developers>
   <scm>
     <connection>scm:https://gitbox.apache.org/repos/asf/cassandra.git</connection>
     <developerConnection>scm:https://gitbox.apache.org/repos/asf/cassandra.git</developerConnection>
     <url>https://gitbox.apache.org/repos/asf?p=cassandra.git;a=tree</url>
   </scm>
+
+  <profiles>
+    <profile>
+      <id>x86_64</id>
+      <activation>
+        <os>
+          <arch>!aarch64</arch>
+        </os>
+      </activation>
+      <dependencies>
```

Comment:
This is the `cassandra-parent` pom, so it should probably be placed under the `<dependencyManagement>` section to avoid any problems for those projects that depend on this pom, but still don't want to see those dependencies in the classpath.
The `<dependency>` probably should be used in the `cassandra-deps-template.xml` which defines the `cassandra-all` artifact. So, the question here is - is it safe putting this dependency in the classpath for all?

Reply:
We have been there. The problem with dependency management is that it can not be specified in profiles. If we put that to dependency management outside profiles in parent, then we can not switch between the platform specific dependencies.